draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/TitleBlockZen.spec.tsx

```diff
@@ -642,6 +643,32 @@ describe("<TitleBlockZen />", () => {
     })
   })
 
+  describe("survey status", () => {
+    it("it doesn't render tag when field is omitted", async () => {
+      render(<TitleBlockZen title="Test Title">Example</TitleBlockZen>)
+
+      await waitFor(() => {
+        expect(
+          screen.queryByTestId("survey-status-tag")
+        ).not.toBeInTheDocument()
+      })
+    })
+    it("it renders tag with correct text and variant when draft status", async () => {
+      render(
+        <TitleBlockZen
+          title="Test Title"
+          surveyStatus={{ text: "draft text", status: "draft" }}
+        >
+          Example
+        </TitleBlockZen>
+      )
+
+      const tagElement = await (await screen.findByTestId("survey-status-tag"))
```

Comment:
this double await seems out of the ordinary, was this intended?

Reply:
it is... the first `await` is for finding the node my test id and the second one os for the firstChild on the next line